docs/development/best-practices.md

```diff
@@ -173,5 +173,12 @@ Use [Named Capturing Groups](https://www.regular-expressions.info/named.html) wh
 
 ### Windows
 
-We recommend you set [`core.autocrlf = input`](https://git-scm.com/docs/gitattributes#_text) in your `gitConfig`, or the carriage return `\r\n` might confuse Renovate bot.
+We recommend you set [`core.autocrlf = input`](https://git-scm.com/docs/gitattributes#_text) in your Git config.
```

Comment:
It's confusing that we explain two ways to set the correct line endings:

1. Run `$ git config --global core.autocrlf input`
2. Add `* text=auto eol=lf` to the `.gitattributes` file

Could we drop one of the explanations? Or do we really need both?

From reading the current docs, it's not clear to me why I may need one or the other.